roles/step_cli/tasks/install.yml

```diff
@@ -37,6 +37,10 @@
         mv /tmp/step_{{ step_cli_version }}/bin/step {{ _step_cli_install_path | d(step_cli_executable) }}
       args:
         executable: /bin/bash
+    # mv does not automatically set selinux labels, so we have to do it ourselves
```

Comment:
We should be able to avoid manual SELinux commands by using `copy` and `file.state = absent` in the commands above.